source/features/ci-link.tsx

```diff
@@ -1,23 +1,24 @@
 import './ci-link.css';
+import oneTime from 'onetime';
 import features from '../libs/features';
 import fetchDom from '../libs/fetch-dom';
 import {getRepoURL} from '../libs/utils';
 import {appendBefore} from '../libs/dom-utils';
 
-let callCount = 0;
-export function getIcon(): Promise<HTMLElement | undefined> {
-	callCount += 1;
-	return fetchDom(`/${getRepoURL()}/commits`, '.commit-build-statuses');
-}
+// Look for the CI icon in the latest 3 days of commits #2884
+export const getIcon = oneTime(fetchDom.bind(null,
+	`/${getRepoURL()}/commits`,
+	'.commit-group:nth-of-type(-n+3) .commit-build-statuses'
```

Comment:
The `3` here means "the last 3 days of commits", that's how commits are grouped in the list. This list could include more than 3 commits and span weeks as well.

<img width="1017" alt="demo" src="https://user-images.githubusercontent.com/1402241/79002398-ad1a3f00-7b50-11ea-8fd9-13216d573710.png">